### PR TITLE
HlA-1059: Reference catalog weights corrected

### PR DESCRIPTION
**Pull request by s-goldman**
This PR corrects the weighting of the reference catalogs; the code now uses 1/sigma**2. The important lines of code are 283 and 284. I also added line 281 to avoid a possible divide by zero error. The remaining changes are the result of code formatting (black).

### CODE REVIEW

File: drizzlepac/haputils/product.py [hunk 1]
```diff
@@ -283,79 +332,132 @@ def align_to_gaia(self, catalog_list=[], output=True, process_type='SVM',
...
+                                    method_name,
+                                    catalog_item,
+                                    ref_catalog,
+                                    fitgeom=mosaic_fitgeom,
+                                )
 
                                 # Define comparison for min cross-matches based on fitgeom used
-                                alignment_pars['determine_fit_quality']['min_xmatches'] = \
-                                    alignment_pars['run_align']['mosaic_fitgeom_list'][mosaic_fitgeom]
+                                alignment_pars["determine_fit_quality"][
```
Comment (mdlpstsci) [inline drizzlepac/haputils/product.py:382]: Same complaint as for Line 355.

File: drizzlepac/haputils/product.py [hunk 2]
```diff
@@ -283,79 +332,132 @@ def align_to_gaia(self, catalog_list=[], output=True, process_type='SVM',
...
+                                    "min_xmatches"
+                                ] = alignment_pars["run_align"]["mosaic_fitgeom_list"][
+                                    mosaic_fitgeom
+                                ]
 
                                 # turn off consistency check for SVM and MVM processing since filter-to-filter
                                 # or visit-to-visit offsets could be large relative to measurement RMS.
-                                alignment_pars['determine_fit_quality']['do_consistency_check'] = False
+                                alignment_pars["determine_fit_quality"][
```
Comment (mdlpstsci) [inline drizzlepac/haputils/product.py:390]: Same complaint as for Line 355.

File: drizzlepac/haputils/product.py [hunk 3]
```diff
@@ -283,79 +332,132 @@ def align_to_gaia(self, catalog_list=[], output=True, process_type='SVM',
...
-                                is_good_fit, _, _, _, _, _ = align.determine_fit_quality_mvm_interface(align_table.imglist,
-                                                                                                       align_table.filtered_table,
-                                                                                                       more_catalogs,
-                                                                                                       num_cat,
-                                                                                                       alignment_pars,
-                                                                                                       print_fit_parameters=True,
-                                                                                                       loglevel=self.log_level,
-                                                                                                       runfile=log_file)
-                                
+                                (
```
Comment (mdlpstsci) [inline drizzlepac/haputils/product.py:399]: Please just revert the text for Lines 399-406 to the unBlack state which is easier to read (IMHO).

**Review by mdlpstsci — CHANGES_REQUESTED** [review]
Since SCSB only has a general set of standards (https://innerspace.stsci.edu/display/SCSB/Code+standards+refresher), and not a specific package we are supposed to use, I have tried to keep my opinion to myself.  However, there are a few places where I found the Black code updates to make the code harder to parse.  Please address these few items.

**Comment by mdlpstsci** [comment]
@s-goldman Where is the update to the "wfpc2_wfpc2_index.json"?

**Comment by mdlpstsci** [comment]
@s-goldman Oops.  I was checking both your PRs and made a comment in the wrong PR.  The change to the parameter index file needs to be in PR#1052 and not this PR for proper tracking of history.  Please remove the changes for config_utils.py from this PR and put into PR#1052.  Once that is done, this PR is ready to go.

**Comment by s-goldman** [comment]
> @s-goldman Oops. I was checking both your PRs and made a comment in the wrong PR. The change to the parameter index file needs to be in PR#1052 and not this PR for proper tracking of history. Please remove the changes for config_utils.py from this PR and put into PR#1052. Once that is done, this PR is ready to go.

I'm sorry, I should have caught that. The change to config_utils is now in the HLA-1052 PR.

**Review by mdlpstsci — APPROVED** [review]
Thanks for all the fixes!

**Pull request merged**